src/pages/OfficeSeats/OfficeSeats.js

```diff
@@ -2,134 +2,109 @@ import React, { Component } from 'react'
 import "./OfficeSeats.css"
 import Dates from '../../components/Dates/Dates'
 import SeatTable from '../../components/SeatTable/SeatTable';
+import axios from "axios"
+
+
 export default class OfficeSeats extends Component {
     constructor () {
         super();
         this.state = {
-            dataHandler:[
-                {
-                    date: 5,
-                    day: "Monday",
-                    month: "Jan"
-                },
-                {
-                    date: 6,
-                    day: "Tuesday",
-                    month: "Jan"
-                },
-                {
-                    date: 7,
-                    day: "Wednesday",
-                    month: "Jan"
-                },
-                {
-                    date: 8,
-                    day: "Thursday",
-                    month: "Jan"
-                },
-                {
-                    date: 9,
-                    day: "Friday",
-                    month: "Jan"
-                },
-                {
-                    nextWeek: true
-                },
-            ],
-            date:null
+            dataHandler: [],      
+            date: null,
+            apiData: ""
         }
     }
+    componentDidMount() {
+        axios.get("http://localhost:8081/week/userWeekData", { headers: { "user-id": "dd" } }
+        ).then((response) => {
+            this.setState({
+                apiData: response.data
+            });
+            this.setCurrentWeekData();
+        })
+            .catch(function (error) {
+                // handle error
+                console.log(error);
+            })
+
+    }
+    weekdate = (year, week, dayNumber) => {
+        var j1 = new Date(year, 0, 10, 12, 0, 0),
+            j2 = new Date(year, 0, 4, 12, 0, 0),
+            mon1 = j2.getTime() - j1.getDay() * 86400000;
+        return new Date(mon1 + ((week - 1) * 7 + (dayNumber - 1)) * 86400000);
+    }
+
```

Comment:
@SuchitaBhosale and @anjali-kapare  use this `weekdate function` to get dates from week count.